01-ruby-fundamentals/debugging-with-pry.md

```diff
@@ -13,11 +13,12 @@ When an exception is raised, it immediately stops the current method, just like
 
 A stack trace contains a ton of useful information, including a list of methods and blocks that Ruby was inside when the exception was raised. Being able to quickly read a complex stack trace is a super useful skill, so let's examine one now.
 
-First, take a look at the following stack trace that came from running some of the tests within the grocery store project.
+First, take a look at the following stack trace that running this new test on the code from the clock example in the [Testing With Minitest](../02-intermediate-ruby/testing-with-minitest.md) lesson.
```

Comment:
This doesn't read as a sentence. "...came from running this new test on the code..." would work. Or feel free to word it your own way